DirectProgramming/DPC++/ProjectTemplates/Hello_World_GPU/README.md

```diff
@@ -0,0 +1,38 @@
+# `Visual Studio based GPU Project` Sample
+
+This project is a template designed to help you create your own Data Parallel C++ application for GPU targets. The template assumes the use of make to build your application. Review the main.cpp source file for help with the header files you should include and how to implement "device selector" code for targeting your application's runtime device
+
+If GPU is not available on your system, you can fallback to cpu or default device.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Windows 10
+| Hardware                          | Integrated Graphics from Intel (GPU) GEN9 or higher
+| Software                          | Intel(R) oneAPI DPC++ Compiler (beta)
+| What you will learn               | Get started with DPC++ for GPU projects
+| Time to complete                  | n/a
+
```

Comment:
This is still an older style of readme, one example is there needs to be a purpose section to the readme, please take a look at https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot as an example

This comment is valid for all the readmes in this PR

Reply:
@JoeOster do you have requirements for the new style of readme files? What sections in your example are mondatory?